apollo-client/src/main/java/com/ctrip/framework/apollo/spring/property/AutoUpdateConfigChangeListener.java

```diff
@@ -89,7 +89,7 @@ public void onChange(ConfigChangeEvent changeEvent) {
     }
   }
 
-  private void updateSpringValue(SpringValue springValue) {
+  protected void updateSpringValue(SpringValue springValue) {
```

Comment:
Why change the method modifiers?

Reply:
AutoUpdateConfigChangeListener can be inherited by other classes, and some methods can be modified to protected, which can be used to implement your own logic. Some people can implement custom annotations and custom parsing, but event listening is required.

Reply:
Since it’s not utilized anywhere in the project, I suggest we consider keeping it private.